modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+
+The function takes a context object as an argument and returns an array of text patterns. This function is called whenever the editor triggers text pattern matching.
+
+For details see xref:content-behavior-options.adoc#text_patterns_lookup[Content Behavior Options].
+
+=== A new promotion element has been added to the default UI.
+
+{productname} 6.2 includes an *Upgrade* promotion button. It appears in the unused corner of the {productname} menu bar.
+
+It can be disabled using the new `promotion` option.
+
+For more information see xref:content-appearance.adoc#premium-upgrade-promotion-option[Premium upgrade promotion option].
+
+=== New `format_noneditable_selector` option to specify the `contenteditable="false"` elements that can be wrapped in a format.
+
+Elements with `contenteditable="false"` attributes can now have formatting applied to them.
+
+And this new capability is configurable using the new `format_noneditable_selector` option.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  format_noneditable_selector: 'span,p.special'
+});
+----
+
+For details see xref:content-formatting.adoc#format_noneditable_selector[Content Formatting].
+
+=== Added `allow` as a valid attribute for the `iframe` element in the editor schema.
+
+Previously, the `allow` attribute was not considered a valid attribute for `iframe` elements in the {productname} schema. It now is.
+
+When a {productname} editor with an `allow` attribute —
+
+[source,html]
+----
+<iframe allow="fullscreen" style="width: 200px; height: 200px; border: 1px solid blue"></iframe>
+----
+
+— is initialized, the `allow` attribute will still be present in the rendered `iframe` element.
+
+=== MenuButtons can now contain a search field
+
+MenuButtons, buttons on the {productname} toolbar that show a menu when clicked, can now present a search field at the top of the menu.
+
+The button refetches items as the search field is updated.
+
+The new xref:mergetags.adoc[Merge Tags] Premium plugin includes a MenuButton with this new search field.
+
+[[improvements]]
+== Improvements
+
+{productname} 6.2 also includes the following improvements:
+
+=== Formatting can now be applied to an element that has a `contenteditable="false"` attribute
+
+Elements with `contenteditable="false"` attributes can now have formatting applied to them.
+
+For example, such elements can now be set to *bold* or _italic_. As well, such elements can have their font and font-size changed.
+
+This new capability is also configurable using the new `format_noneditable_selector` option.
+
+For details see xref:content-formatting.adoc#format_noneditable_selector[Content Formatting].
+
+=== The autocompleter now supports a multiple character trigger using the new trigger configuration.
+
+Previously, the autocompleter only supported a single character trigger.
```

Comment:
Do we mention anything about deprecation in the release notes? Or not typically?